### DISCUSSION TITLE
Is it compatible with Delphi5?

### DISCUSSION THREAD

**Discussion by Francisko68** [landrix/TPrintPreview-for-Delphi#9] [General]
I already had to make some modifications to version 5.93 to make it work with my Delphi 5, but this version 5.95 gives me a "long line error more than 1053 characters"and I can't see where the problem is. I already know that Delphi5 is very old, but my software is developed there and we have a legal license for the program ... Thanks for any help.

**Comment by landrix** [comment]
there are no plans for Delphi 5
i think you should stay on http://www.delphiarea.com/products/delphi-components/preview/

**Comment by landrix** [comment]
You can send me your code and I will make a compare.

**Reply by Francisko68** [reply]
Ok, I am attaching the modified code that works correctly in Delphi 5; It is version 5.93.

[Preview593,pas.zip](https://github.com/landrix/TPrintPreview-for-Delphi/files/6157303/Preview593.pas.zip)

On the other hand, the "long line error ..." error was due to the encoding of the file, I have passed it to ansi and it no longer gives that error. But obviously it does not work ... many {$ IFDEF COMPILERx_UP} are missing and of course functions that did not exist in D5 are used. Thanks for your help!
